core/iwasm/include/wasm_export.h

```diff
@@ -800,6 +800,27 @@ wasm_runtime_call_wasm_v(wasm_exec_env_t exec_env,
                          uint32_t num_results, wasm_val_t results[],
                          uint32_t num_args, ...);
 
+/**
+ * Call a function reference of a given WASM runtime instance with
+ * arguments.
```

Comment:
Could we mention that it may be unsafe since the function is not wasm-wise exported explicitly?

Reply:
done